arch/arm64/boot/dts/qcom/msm8953.dtsi

```diff
@@ -2608,6 +2622,22 @@
 			qcom,controlled-remotely;
 		};
 
+		blsp2_uart0: serial@7aef000 {
```

Comment:
blsp2_uart1 like the clock name GCC_BLSP2_UART1_APPS_CLK

Reply:
```

		uart_0: serial@78af000 {
			compatible = "qcom,msm-uartdm-v1.4", "qcom,msm-uartdm";
			reg = <0x078af000 0x200>;
			interrupts = <GIC_SPI 107 IRQ_TYPE_LEVEL_HIGH>;
			clocks = <&gcc GCC_BLSP1_UART1_APPS_CLK>,
				 <&gcc GCC_BLSP1_AHB_CLK>;
			clock-names = "core", "iface";
			status = "disabled";
		};
```

Reply:
This also should called as blsp1_uart1 sadly it was named uart_0.

Reply:
Good catch!

I adapted this from downstream: https://git.codelinaro.org/clo/la/kernel/msm-4.9/-/blob/LA.UM.10.6.2.c26/arch/arm64/boot/dts/qcom/msm8953.dtsi?ref_type=heads#L612

With a bit more experience now, I'm finding that `blsp2_uart0` should probably actually be `blsp_uart5`: https://git.codelinaro.org/clo/la/kernel/msm-4.9/-/blob/LA.UM.10.6.2.c26/arch/arm64/boot/dts/qcom/msm8953-pinctrl.dtsi#L608
This is also what the pins used downstream for `blsp2_uart0` are mapped to in [`drivers/pinctrl/qcom/pinctrl-msm8953.c`](https://github.com/msm8953-mainline/linux/blob/070c4019af1a4649f3268642c756f185fc5ecc8c/drivers/pinctrl/qcom/pinctrl-msm8953.c#L752) upstream.

I'll rewrite this commit using blsp_uart5 instead, as that is what MSM8916 and MSM8939 do upstream.